### PR TITLE
Bah 8940 Hospital Info Form

### PR DESCRIPTION
**Pull request by imeecuison**
## Description
Fields that capture hospital name, address, and dates of hospitalization.
http://localhost:3001/disability-benefits/apply/form-526-all-claims/hospitalization-history

## Testing done
Unit testing on:
tests/pages/hospitalizationhistory.unit.spec.jsx

## Screenshots

screen shot 2018-11-07 at 10 36 56 am
screen shot 2018-11-07 at 10 37 04 am

## Acceptance criteria
- [ ] The user is able to populate the following:
- [ ] Hospital’s name { text field }
- [ ] Hospital’s address { address pattern }
- [ ] From { Date }
- [ ] To { Date}
- [ ] The user is able to add additional Hospitalization information ("add another...")
- [ ] The user is able to move to the next step in the process, Story#14532: Form 21-8940: Ability to populate Disability Dates interview questions

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/disability-benefits/all-claims/tests/pages/hospitalizationhistory.unit.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
+import React from 'react';
```
Comment (aaronjlech) [inline src/applications/disability-benefits/all-claims/tests/pages/hospitalizationhistory.unit.spec.jsx]: can we camelCase this file name? `hospitalizationHistory`

File: src/applications/disability-benefits/all-claims/pages/hospitalizationHistory.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    items: {
+      hospitalizationtDateRange: dateRangeUI(
+        'From',
+        'To',
+        'End of hospitalization must be after start of treatment',
+      ),
+      hospitalName: {
+        'ui:title': 'Name of hospital',
+      },
+      streetAddress: {
```
Comment (aaronjlech) [inline src/applications/disability-benefits/all-claims/pages/hospitalizationHistory.js]: Is there anyway we can use the `addressUI()` here and omit the pieces we don't need(like Country).

**Review by aaronjlech — CHANGES_REQUESTED** [review]

**Comment by aaronjlech** [comment]
image
Something weird going on here...

**Review by aaronjlech — COMMENTED** [review]
This all looks good! I think your unit tests are breaking due to this change though.

File: src/applications/disability-benefits/all-claims/tests/pages/hospitalizationhistory.unit.spec.jsx (hunk 1 above)
Comment (imeecuison) [reply src/applications/disability-benefits/all-claims/tests/pages/hospitalizationhistory.unit.spec.jsx]: changed!

File: src/applications/disability-benefits/all-claims/pages/hospitalizationHistory.js (hunk 2 above)
Comment (imeecuison) [reply src/applications/disability-benefits/all-claims/pages/hospitalizationHistory.js]:
```
  hospitalAddress: {
            ...address,
            required: [],
            properties: omit(address.properties, ['addressLine3', 'country']),
          },
```
This is what I did. ^^

**Comment by imeecuison** [comment]
@aaronjlech : Can you check on these changes?

**Review by aaronjlech — APPROVED** [review]
looks good!

**Review by devin-mccurdy — APPROVED** [review]

**Review by rianfowler — APPROVED** [review]

**Comment by imeecuison** [comment]
Hey @jbalboni and @cvalarida ! When you have a minute, can you take a look at this PR? I need a second approval. Thank you!

**Review by jbalboni — APPROVED** [review]

**Pull request merged**